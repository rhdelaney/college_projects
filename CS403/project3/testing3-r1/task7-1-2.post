(println "trying (twinPrimes 12)")
(stream-display 243 (twinPrimes 12))
(println)
