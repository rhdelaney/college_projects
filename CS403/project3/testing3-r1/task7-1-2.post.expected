trying (twinPrimes 12)
((2 3) (2 5) (2 7) (2 11) (2 13) (3 5) (3 7) (3 11) (3 13) (5 7) (5 11) (5 13) (5 17) (7 11) (7 13) (7 17) (7 19) (11 13) (11 17) (11 19) (11 23) (13 17) (13 19) (13 23) (17 19) (17 23) (17 29) (19 23) (19 29) (19 31) (23 29) (23 31) (29 31) (29 37) (29 41) (31 37) (31 41) (31 43) (37 41) (37 43) (37 47) (41 43) (41 47) (41 53) (43 47) (43 53) (47 53) (47 59) (53 59) (53 61) (59 61) (59 67) (59 71) (61 67) (61 71) (61 73) (67 71) (67 73) (67 79) (71 73) (71 79) (71 83) (73 79) (73 83) (79 83) (79 89) (83 89) (89 97) (89 101) (97 101) (97 103) (97 107) (97 109) (101 103) (101 107) (101 109) (101 113) (103 107) (103 109) (103 113) (107 109) (107 113) (109 113) (127 131) (127 137) (127 139) (131 137) (131 139) (137 139) (137 149) (139 149) (139 151) (149 151) (149 157) (151 157) (151 163) (157 163) (157 167) (163 167) (163 173) (167 173) (167 179) (173 179) (173 181) (179 181) (179 191) (181 191) (181 193) (191 193) (191 197) (191 199) (193 197) (193 199) (197 199) (199 211) (211 223) (223 227) (223 229) (223 233) (227 229) (227 233) (227 239) (229 233) (229 239) (229 241) (233 239) (233 241) (239 241) (239 251) (241 251) (251 257) (251 263) (257 263) (257 269) (263 269) (263 271) (269 271) (269 277) (269 281) (271 277) (271 281) (271 283) (277 281) (277 283) (281 283) (281 293) (283 293) (307 311) (307 313) (307 317) (311 313) (311 317) (313 317) (331 337) (337 347) (337 349) (347 349) (347 353) (347 359) (349 353) (349 359) (353 359) (359 367) (367 373) (367 379) (373 379) (373 383) (379 383) (379 389) (383 389) (389 397) (389 401) (397 401) (397 409) (401 409) (409 419) (409 421) (419 421) (419 431) (421 431) (421 433) (431 433) (431 439) (431 443) (433 439) (433 443) (439 443) (439 449) (443 449) (449 457) (449 461) (457 461) (457 463) (457 467) (461 463) (461 467) (463 467) (467 479) (479 487) (479 491) (487 491) (487 499) (491 499) (491 503) (499 503) (499 509) (503 509) (509 521) (521 523) (541 547) (547 557) (557 563) (557 569) (563 569) (563 571) (569 571) (569 577) (571 577) (577 587) (587 593) (587 599) (593 599) (593 601) (599 601) (599 607) (601 607) (601 613) (607 613) (607 617) (607 619) (613 617) (613 619) (617 619) (619 631) (631 641) (631 643) (641 643) (641 647) (641 653) (643 647) (643 653) (647 653) (647 659) ...)
